src/core/Elsa.Core/Activities/ControlFlow/ForEach/ForEach.cs

```diff
@@ -24,8 +24,8 @@ public class ForEach : Activity
             DefaultSyntax = SyntaxNames.Json,
             SupportedSyntaxes = new[] { SyntaxNames.Json, SyntaxNames.JavaScript, SyntaxNames.Liquid }
         )]
-        public ICollection<object> Items { get; set; } = new Collection<object>();
-        
+        public IEnumerable Items { get; set; } = new Collection<object>();
```

Comment:
Would be good to add a comment here explaining why we're using `IEnumerable` (and reference this PR).

Reply:
sure - i update the PR

Reply:
done :)